### ISSUE TITLE
Consider using one export

### ISSUE THREAD

**Issue by vladshcherbin** [scriptcoded/sql-highlight#5]
👋 

Hey, since this library has only one function, maybe it's a good idea to export it like this:

```js
module.exports = highlight;
```

It will be possible to use imports this way:

```js
const highlight = require('sql-highlight');
// or
import highlight from 'sql-highlight';
```

Using `{ highlight }` in imports gives me false expectations that there are more functions inside while there is only one.

What do you think?

**Comment by scriptcoded** [comment]
I think that would make sense. I don't plan to add new features to this library any time soon, so the API shouldn't be changing further in the future. The question is if this change should be backwards compatible, ie. supplying both a default and named export. 

I think something like the following might be in order
```javascript
module.exports = highlight
module.exports.highlight = highlight
```

Any thoughts?

**Comment by vladshcherbin** [comment]
@scriptcoded I think, it's perfectly fine to make it backwards compatible and deprecate or even change to ES module export in next major version 😉

**Comment by scriptcoded** [comment]
@vladshcherbin :+1: Sounds good. I'll see if I get time to do this tomorrow.

**Comment by scriptcoded** [comment]
Resolved in 80bb46617704e87c2d4cfa54ccb7ecd8dab1a486. I'll make sure to release this later today.
